.github/workflows/build.yml

```
@@ -0,0 +1,20 @@
+  build:
+    name: Try to build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install modules
+        run: yarn install --frozen-lockfile
+      - name: Run build
+        run: yarn build
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)